charts/intel-gpu-resource-driver/templates/gpu-resource-driver.yaml

```diff
@@ -0,0 +1,235 @@
+---
+apiVersion: gpu.resource.intel.com/v1alpha2
+kind: GpuClassParameters
+metadata:
+  name: intel-gpu-params-shared
+  labels:
+    {{- include "intel-gpu-resource-driver.labels" . | nindent 4 }}
+spec:
+  shared: true
+---
+apiVersion: resource.k8s.io/v1alpha2
+kind: ResourceClass
+metadata:
+  name: intel-gpu-shared
+driverName: gpu.resource.intel.com
+parametersRef:
+  apiGroup: gpu.resource.intel.com/v1alpha2
+  kind: GpuClassParameters
+  name: intel-gpu-params-shared
+
+---
+apiVersion: gpu.resource.intel.com/v1alpha2
+kind: GpuClassParameters
+metadata:
+  name: intel-gpu-params
+spec:
+  shared: false
+---
+apiVersion: resource.k8s.io/v1alpha2
+kind: ResourceClass
+metadata:
+  name: intel-gpu
+driverName: gpu.resource.intel.com
+parametersRef:
+  apiGroup: gpu.resource.intel.com/v1alpha2
+  kind: GpuClassParameters
+  name: intel-gpu-params
+
+---
+apiVersion: gpu.resource.intel.com/v1alpha2
+kind: GpuClassParameters
+metadata:
+  name: intel-gpu-monitor-params
+spec:
+  monitor: true
+  shared: false
+---
+apiVersion: resource.k8s.io/v1alpha2
+kind: ResourceClass
+metadata:
+  name: intel-gpu-monitor
+driverName: gpu.resource.intel.com
+parametersRef:
+  apiGroup: gpu.resource.intel.com/v1alpha2
+  kind: GpuClassParameters
+  name: intel-gpu-monitor-params
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: defaults
+  namespace: {{ include "intel-gpu-resource-driver.namespace" . }}
+data:
+# It is possible to override driver hardcoded defaults:
+# - resourceClaimParameters:
+#     if resource claim did not include any parameters, these will be used
+# - vf-memory
+#     per product amount of local memory, in MiB, the auto-added VFs will get
+  resourceClaimParameters.config: |
+    count=1
+    type=gpu
+    shareable=true
+  vf-memory.config: |
+    {
+      "max1550": 16385,
+      "max1450": 16384,
+      "max1100": 8192,
+      "flex140": 2048,
+      "flex170": 4096,
+    }
```

Comment:
Are these something that should be configurable via values?

Reply:
Not necessarily, this ConfigMap is read often, so once cluster admin changes it, there's even no need to redeploy anything, so can be done post-installation.